scripts/dovecot/build-obs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Install dependencies
+echo "Installing dependencies for this script:"
+sudo apt install devscripts build-essential osc curl git debhelper-compat
```

Comment:
```suggestion
echo "Installing dependencies for this script:"
sudo apt install -y devscripts build-essential osc curl git debhelper-compat
```

This doesn't need to be interactive, no?

Reply:
true